ROADMAP.md

```diff
@@ -10,21 +10,30 @@ This document defines the roadmap for TiDB development.
 ## TiDB:
 
 - [ ] Optimizer
-  - [ ] Refactor Ranger
-  - [ ] Optimize the statistics info 
+  - [x] Refactor Ranger
   - [ ] Optimize the cost model
+  - [ ] Join Reorder
+- [ ] Statistics
+  - [x] Update statistics dynamically according to the query feedback
+  - [x] Analyze table automatically
+  - [ ] Improve the accuracy of Row Cout estimation
 - [ ] Executor
+  - [ ] Pushdown the Projection operator to the Coprocessor
```

Comment:
Pushdown -> Push down

Note: Use the two words version as a verb. "pushdown" is usually used as a noun.